libs/ui/src/hooks/use_screen_info.ts

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+
+/** Data type for {@link useScreenInfo}. */
+export interface ScreenInfo {
+  readonly isPortrait: boolean;
+}
+
+function isScreenPortrait(): boolean {
+  return window.innerHeight > window.innerWidth;
+}
+
+/**
+ * React hook to get info about the current screen and viewport.
+ */
+export function useScreenInfo(): ScreenInfo {
+  const [screenInfo] = useState<ScreenInfo>({
+    isPortrait: isScreenPortrait(),
+  });
+
+  // TODO: Add a debounced window resize listener and update screen info accordingly.
```

Comment:
Would this just be for development?

Reply:
(the listener)

Reply:
Yeah, just dev - I've been dragging my window between my ELO and my laptop to test portrait and landscape, so more of a quality-of-life improvement for me 😅